db/migrate/20171212084905_change_game_mode_and_player_types_null.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeGameModeAndPlayerTypesNull < ActiveRecord::Migration[5.1]
+  def change
+    change_column :events, :player_type, :integer, :null => true
+    change_column :events, :game_mode, :integer, :null => true
```

Comment:
If you end up saving it in the same column, there is no need for this in the migration and also we obviously have to figure out if player_type is gonna stay or not.